pkg/clustermesh/kvstoremesh/remote_cluster.go

```diff
@@ -259,9 +274,13 @@ func (rc *remoteCluster) Status() *models.RemoteCluster {
 		Identities: rc.identities.watcher.Synced(),
 		Endpoints:  rc.ipcache.watcher.Synced(),
 	}
+	if status.Config != nil && status.Config.ServiceExportsEnabled != nil {
+		status.Synced.ServiceExports = ptr.To(rc.serviceExports.watcher.Synced())
+	}
```

Comment:
I refactored a bit further this as well with the `ServiceExports` field that can now be nil, I think leaving this to nil unless we actually sync the service exports (remote cluster supports that) is more explicit and makes the logic slightly simpler.